provider/identity_provider_resource.go

```diff
@@ -93,6 +98,28 @@ func (t *IdentityProviderResourceType) htpasswdSchema() tfsdk.NestedAttributes {
 	})
 }
 
+func (t *IdentityProviderResourceType) gitlabSchema() tfsdk.NestedAttributes {
+	return tfsdk.SingleNestedAttributes(map[string]tfsdk.Attribute{
+		"ca": {
```

Comment:
please add a description for each nr of the attributes

Reply:
Descriptions added.